File: library/src/main/java/org/jellyfin/apiclient/logging/NullLogger.java
package org.jellyfin.apiclient.logging;

/**
 * ILogger implementation that does nothing
 */
public class NullLogger implements ILogger {
    @Override
    public final void info(String formatString, Object... paramList) {
        // Do nothing
    }

    @Override
    public final void error(String formatString, Object... paramList) {
        // Do nothing
    }

    @Override
    public final void debug(String formatString, Object... paramList) {
        // Do nothing
    }

    @Override
    public final void exception(String formatString, Exception exception, Object... paramList) {
        // Do nothing
    }
}

File: library/src/main/java/org/jellyfin/apiclient/logging/NullLogger.kt
package org.jellyfin.apiclient.logging

class NullLogger : ILogger {
	override fun debug(str: String, vararg args: Any) = Unit
	override fun info(str: String, vararg args: Any) = Unit
	override fun error(str: String, vararg args: Any) = Unit
	override fun error(str: String, exception: Exception, vararg args: Any) = Unit
}
